build.sbt

```diff
@@ -34,9 +34,9 @@ ThisBuild / resolvers += "Gemini Repository".at(
   "https://github.com/gemini-hlsw/maven-repo/raw/master/releases"
 )
 
-// This key is used to find the JRE dir. It could/should be overridden on a user basis
-// Add e.g. a `jres.sbt` file with your particular configuration
-ThisBuild / ocsJreDir := Path.userHome / ".jres11"
+// This key is used to find the JRE dir that will be bundled with packaged deployments.
+// It could/should be overridden on a user basis. E.g. add a `jres.sbt` file with your particular configuration.
+ThisBuild / packagedJreDir := Path.userHome / ".jres" / "linux" / "jdk-11.0.21+9-jre"
```

Comment:
I wonder if we should go 17

Reply:
In the past we had a link like `.jres/linux/JDK` wheere you locally select the JVM changing the link. WDYT?

Reply:
That would be fine. Could you add a note in the README?